build_runner/lib/src/daemon/change_providers.dart

```diff
@@ -17,21 +22,18 @@ class AutoChangeProvider implements ChangeProvider {
   Future<List<WatchEvent>> collectChanges() async => [];
 }
 
-// TODO(grouma) - collect changes through a one time file scan instead of
-// buffering changes from a change stream. This will have better performance
-// on Windows.
```

Comment:
I think there is a tracking issue that should be linked.

Reply:
I can't find one, if you do please close and link it